core/templates/dev/head/domain/exploration/SolutionObjectFactory.js

```diff
@@ -68,8 +68,7 @@ oppia.factory('SolutionObjectFactory', [
       var explanation = (
         $filter('convertToPlainText')(this.explanation));
       return (
-        '[' + solutionType + ' solution is ' + correctAnswer + '] ' +
-        explanation);
+        solutionType + ' solution is ' + correctAnswer + '. ' + explanation);
```

Comment:
Since it is a list it has braces and quotation marks. I could create specific summaries based on interactions in the SolutionObjectFactory.getSummary function.